.github/workflows/build-and-push-cliain.yml

```diff
@@ -6,12 +6,7 @@ on:
     branches:
       - 'main'
     paths:
```

Comment:
Why not remove the `paths` entry entirely?

Reply:
Right, will remove

Reply:
done